tests/Browser/AuthenticationTest.php

```diff
@@ -35,6 +35,48 @@ public function testLoginAndLogout()
         $this->removeUser($user->id);
     }
 
+    /**
+     * Testing the register form
+     *
+     * @return void
+     */
+    public function testRegisterForm()
+    {
+        $user_created = false;
+
+        $name = 'Mr Bean';
+        $username = 'MrBean';
+        $emailRegister = 'mr.bean@gmail.com';
+        $passwordRegister = 'ABC123doremi29!';
+        $passwordRegister_confirmation = $passwordRegister;
```

Comment:
In order to respect PHP coding style these variables should be snake_case:
`$email_register`
`$password_register`
`$password_register_confirmation`
And will need to be changed in the entire document